docs/user-guide/disassembler-backends/ghidra.md

```diff
@@ -2,30 +2,39 @@
 
 ## Install
 
-### Docker
```

Comment:
Can we add one sentence explaining there are two ways to install, to set up the tabbed options?